extra_data/components.py

```diff
@@ -1446,6 +1446,9 @@ class JUNGFRAU(MultimodDetectorBase):
     n_modules: int
       Number of detector modules in the experiment setup. Default is
       None, in which case it will be estimated from the available data.
+    st_modno: int
+      The module number that the detector starts with.
```

Comment:
```suggestion
      The module number in the source name  for the first detector module.
```

Reply:
Thank you. I updated the description. [9897575bd59989213d70931f09dde5c67d390841](https://github.com/European-XFEL/EXtra-data/pull/379/commits/9897575bd59989213d70931f09dde5c67d390841)